### PR TITLE
Use rubocop-shopify gem

### PR DESCRIPTION
**Pull request by andyw8**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2020-10-05 19:12:07 UTC using RuboCop version 0.92.0.
```
Comment (andyw8) [inline .rubocop_todo.yml:3]: I re-generated this. The latest rubocop-shopify depends on ~> 0.89.0, hence the lower version.

File: .rubocop.yml [hunk 2]
```diff
@@ -1,5 +1,7 @@
+inherit_gem:
+  rubocop-shopify: rubocop.yml
```
Comment (andyw8) [inline .rubocop.yml:2]: As recommended in https://github.com/shopify/ruby-style-guide

**Review by paulomarg — APPROVED** [review]
LGTM, I failed to see that before when I added rubocop back to this repo.

File: test/location_test.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@ class LocationTest < Test::Unit::TestCase
   test '#inventory_levels returns all inventory_levels associated with this location' do
     location = ShopifyAPI::Location.new(id: 487838322)
     expected_body = JSON.parse(load_fixture('inventory_levels'))
-    expected_body['inventory_levels'].delete_if {|level| level['location_id'] != location.id }
+    expected_body['inventory_levels'].delete_if { |level| level['location_id'] != location.id }
```
Comment (andyw8) [inline test/location_test.rb:7]: For some reason, this wasn't picked up when re-generating `.rubocop_todo.yml`

**Pull request merged**